logs/logs.log

```diff
@@ -0,0 +1,39 @@
+21-09-2020 00:13:45.814 [task-1] INFO  com.zaxxer.hikari.HikariDataSource.getConnection - HikariPool-1 - Start completed.
+21-09-2020 00:13:45.838 [task-1] INFO  org.hibernate.dialect.Dialect.<init> - HHH000400: Using dialect: org.hibernate.dialect.PostgreSQLDialect
+21-09-2020 00:13:46.247 [main] INFO  o.s.b.w.e.tomcat.TomcatWebServer.start - Tomcat started on port(s): 8080 (http) with context path ''
+21-09-2020 00:13:46.249 [main] INFO  s.d.s.w.p.DocumentationPluginsBootstrapper.start - Context refreshed
+21-09-2020 00:13:46.284 [main] INFO  s.d.s.w.p.DocumentationPluginsBootstrapper.start - Found 1 custom documentation plugin(s)
+21-09-2020 00:13:46.329 [main] INFO  s.d.s.w.s.ApiListingReferenceScanner.scan - Scanning for api listing references
+21-09-2020 00:13:46.595 [main] INFO  o.s.d.r.c.DeferredRepositoryInitializationListener.onApplicationEvent - Triggering deferred initialization of Spring Data repositories…
+21-09-2020 00:13:47.839 [task-1] INFO  o.h.e.t.j.p.i.JtaPlatformInitiator.initiateService - HHH000490: Using JtaPlatform implementation: [org.hibernate.engine.transaction.jta.platform.internal.NoJtaPlatform]
+21-09-2020 00:13:47.847 [task-1] INFO  o.s.o.j.LocalContainerEntityManagerFactoryBean.buildNativeEntityManagerFactory - Initialized JPA EntityManagerFactory for persistence unit 'default'
+21-09-2020 00:13:48.415 [main] INFO  o.s.d.r.c.DeferredRepositoryInitializationListener.onApplicationEvent - Spring Data repositories initialized!
+21-09-2020 00:13:48.424 [main] INFO  c.e.t.ThunderboltApplication.logStarted - Started ThunderboltApplication in 6.98 seconds (JVM running for 7.995)
+21-09-2020 00:13:59.270 [http-nio-8080-exec-2] INFO  o.a.c.c.C.[Tomcat].[localhost].[/].log - Initializing Spring DispatcherServlet 'dispatcherServlet'
+21-09-2020 00:13:59.271 [http-nio-8080-exec-2] INFO  o.s.web.servlet.DispatcherServlet.initServletBean - Initializing Servlet 'dispatcherServlet'
+21-09-2020 00:13:59.279 [http-nio-8080-exec-2] INFO  o.s.web.servlet.DispatcherServlet.initServletBean - Completed initialization in 8 ms
+21-09-2020 00:31:27.280 [http-nio-8080-exec-1] INFO  c.e.t.controller.ProductsController.getAllProductDetails - Inside Get All Product Details
+21-09-2020 02:38:02.061 [SpringContextShutdownHook] INFO  o.s.o.j.LocalContainerEntityManagerFactoryBean.destroy - Closing JPA EntityManagerFactory for persistence unit 'default'
+21-09-2020 02:38:02.076 [SpringContextShutdownHook] INFO  o.s.s.c.ThreadPoolTaskExecutor.shutdown - Shutting down ExecutorService 'applicationTaskExecutor'
+21-09-2020 02:38:02.078 [SpringContextShutdownHook] INFO  com.zaxxer.hikari.HikariDataSource.close - HikariPool-1 - Shutdown initiated...
+21-09-2020 02:38:02.091 [SpringContextShutdownHook] INFO  com.zaxxer.hikari.HikariDataSource.close - HikariPool-1 - Shutdown completed.
```

Comment:
Never commit a log file. Delete this one and add *.log to gitignore to prevent further additions.

Reply:
Can't it be off to some reference atleast?

Reply:
You can have references separately, but not as a part of the codebase